addon/components/sl-select.js

```diff
@@ -116,20 +133,71 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, {
     setupSelect2: Ember.on(
         'didInsertElement',
         function() {
-            const input = this.$( 'input' ).select2({
+            const input = this.$( 'input' ).select2( this.get( 'options' ) );
+
+            input.on( 'change', () => {
+                this.set( 'value', input.select2( 'val' ) );
+            });
+
+            const originalBodyOverflow = document.body.style.overflow || 'auto';
+
+            input.on( 'select2-open', () => {
+                document.body.style.overflow = 'hidden';
+            });
+
+            input.on( 'select2-close', () => {
+                document.body.style.overflow = originalBodyOverflow;
+            });
+
+            this.$( 'input.select2-input' ).attr(
```

Comment:
Is this call even necessary with a/the `placeholder` property being passed to the select2 initializer as an option?